src/it/includes-excludes/src/test/java/org/apache/maven/it0055/PersonTest.java

```diff
@@ -18,14 +18,10 @@
  */
 package org.apache.maven.it0055;
 
-import junit.framework.TestCase;
-
-public class PersonTest extends TestCase {
+public class PersonTest {
     public void testPerson() {
         Person person = new Person();
 
         person.setName("foo");
-
-        assertEquals("foo", person.getName());
```

Comment:
We could keep those lines after the proposed addition of `import static org.junit.jupiter.api.Assertions.assertEquals`.